src/main/java/net/dv8tion/jda/core/managers/impl/PresenceImpl.java

```diff
@@ -231,7 +231,7 @@ private JSONObject getGameJson(Game game)
         JSONObject gameObj = new JSONObject();
         gameObj.put("name", game.getName());
         gameObj.put("type", game.getType().getKey());
-        if (game.getType() == Game.GameType.TWITCH && game.getUrl() != null)
+        if (game.getUrl() != null)
             gameObj.put("url", game.getUrl());
```

Comment:
Does it do anything if we send a url when type is not STREAMING? (Game with url but DEFAULT type is created, when specifying a invalid streaming url to `Game.of(String, String)`)